OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,20 +1,201 @@
+For this setup we will use a user defined network using bridge driver.
+
+Create a user defined network using the bridge driver by executing the following command :
+
+    $ docker network create -d bridge <some name>
+   
+Sample command:
+
+    $ docker network create -d bridge SOANet
+
+
+# Mount a host directory as a data volume
+
+Data volumes are designed to persist data, independent of the container’s lifecycle. The default location of the volume in container is under `/var/lib/docker/volumes`. There is an option to mount a directory from the host into a container as volume. In this project we will use that option for the data volume. The volume will be used to store Database datafiles and Weblogic server domain files.This volume will be created on the host at `/scratch/DockerVolume/SOAVolume/`. Since the volume is created as "root" user, provide read/write/execute permissions to "oracle" user (by providing permissions to "others"), as all operations inside the container happens with "oracle" user login.
+
+    $ chmod -R 777 `/scratch/DockerVolume/SOAVolume`
+
+**Database**
+
+You need to have a running database container or a database running on any machine. The database connection details are required for creating SOA specific RCU schemas while configuring SOA domain. While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
+
+Please refer README.md under docker/OracleDatabase for details on how to build Oracle Database image.
+https://github.com/oracle/docker-images/tree/master/OracleDatabase
+
+
+## Running Oracle Database in a Docker container
+
+Create an environment file db.env.list
+
+    ORACLE_SID=<DB SID>
+    ORACLE_PDB=<PDB ID>
+    ORACLE_PWD=<password>
+    
+Sample Data will look like this:
+
+    ORACLE_SID=soadb
+    ORACLE_PDB=soapdb
+    ORACLE_PWD=Welcome1
+
+Sample Command to Start the Database is as follows
+
+     $ docker run --name soadb  --network=SOANet -p 1521:1521 -p 5500:5500 -v /scratch/DockerVolume/SOAVolume/DB:/opt/oracle/oradata --env-file ./db.env.list  oracle/database:12.2.0.1-ee
+     
+The above command starts a DB container attaching to a network and mounting a host directory as `/opt/oracle/oradata` for persistence. 
+It maps the containers 1521 and 5500 port to respective host port such that the services can be accessible outside of localhost.
+
+
+## SOA Distributable version 12.2.1.3 Docker image Creation and Running
+
+To build a SOA image either you can start from building Oracle JDK and Oracle FMW Infrastrucure image or use the already available Oracle FMW Infrastructure image. The FMW Infrastructure image is available in the Oracle Container Registry and can be pulled from there. If you plan to use the Oracle FMW Infrastructure image from the Oracle Container Registry, you can skip the next two steps and continue with "Building a Docker Image for SOA".
```

Comment:
The build instructions create an image with a slightly different namespace/repo combination. You may want to mention how to re-tag the image from the Oracle Container Registry to match the `FROM` line in your `Dockerfile`.

Reply:
Please create a Markdown link to the [Oracle Container Registry](https://container-registry.oracle.com).